CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 All notable changes to teamcity-rest-client library will be documented in this file.
+
+## [1.12.2] - 2020-10-16
```

Comment:
We use semantic versioning so it's better to increase minor version, not patch version when you add new API, see [CONTRIBUTING.MD](https://github.com/JetBrains/teamcity-rest-client/blob/master/CONTRIBUTING.md).

Reply:
Ops. For some reason `12` is a major version for me

Reply:
Ok, next time...